src/components/hero/_hero.scss

```diff
@@ -98,6 +108,22 @@
         height: 100%;
     }
 
+    &__information {
+        display: flex;
+        flex-wrap: wrap;
+        column-gap: 1rem;
+
+        & .ons-hero__panel {
+            padding: 0.5rem;
```

Comment:
Tiny nitpick: based on the design I think this should be (gives a bit more breathing room on mobile especially)

```suggestion
            padding: 1rem 0.5rem;
```

Reply:
I think what I already has matches the figma. Its 8px top and bottom. Although I can see there is a red panel image in the ticket that has more space. Ill double check this

<img width="1377" alt="Screenshot 2025-04-01 at 12 30 47" src="https://github.com/user-attachments/assets/cb046293-57e0-45fd-bfe5-b8986dc246d7" />

Reply:
Oh actually did you mean to suggest `padding: 0.5rem 1rem;`?

Reply:
Actually no I think I was looking at the larger red example as well - but you're right, this should be `1rem 0.5rem`.  Looking good now, thank you :)